packages/substrate/src/utils/deposit/deposit.ts

```diff
@@ -0,0 +1,95 @@
+import type { XcmMultiAssetIdType } from '@buildwithsygma/core';
+import { Environment } from '@buildwithsygma/core';
+import type { ApiPromise, SubmittableResult } from '@polkadot/api';
+import type { SubmittableExtrinsic } from '@polkadot/api-base/types';
+import { numberToHex } from '@polkadot/util';
+
+/*
+ * @category Bridge deposit
+ * @param {Environment} environment - The environment from which the transfer is being made
+ * @param {string} address - The recipient address.
+ * @param {string} domainId - The domain identifier.
+ * @returns {object} - The destination multilocation object.
+ */
+export const createDestIdMultilocationData = (address: string, domainId: string): object => {
+  const environment = process.env.SYGMA_ENV;
+
+  switch (environment) {
```

Comment:
@saadjhk is this compatible with Tangle?

Reply:
Im testing this

Reply:
Validated here https://chainsafeio.slack.com/archives/C02KNM76NR4/p1718896504214529?thread_ts=1718723931.721569&cid=C02KNM76NR4